spec/requests/providers/proceeding_merits_task/is_client_child_subject_controller_spec.rb

```diff
@@ -73,10 +73,9 @@ module ProceedingMeritsTask
             expect(legal_aid_application.legal_framework_merits_task_list).to have_not_started_task(:PB003, :client_relationship_to_proceeding)
           end
 
-          # TODO: change the following to redirect to question 4 when it is added
           it "redirects to merits task list page" do
```

Comment:
should it be it "redirects to check who client is" page?

Reply:
Yep, I've made the change